doc/man1/openssl-list.pod.in

```diff
@@ -71,13 +73,17 @@ Display a list of standard commands.
 
 =item B<-digest-commands>
 
+This option is deprecated.
```

Comment:
Please state what users should use instead

Reply:
Fixed. Thanks!